src/system/storage/storage.h

```diff
@@ -0,0 +1,64 @@
+/*----------------------------------------------------------------------*
+ *  _   __ _     _____ _____                                            *
+ * | | / /| |   |_   _/  __ \   This software is been developed by      *
+ * | |/ / | |     | | | /  \/   a group of enthusiast hobbiest          *
+ * |    \ | |     | | | |       with the purpose of learn and           *
+ * | |\  \| |_____| |_| \__/\   have fun, so nobody is responsible or   *
+ * \_| \_/\_____/\___/ \____/   will provide warranty.                  *
+ *                                                                      *
+ * This software will run in a ESP8266 microcontrolller, the objective  *
+ * is to have a irrigation system controler that can read some sensors  *
+ * and decide if a valve should be open.                                *
+ * There is not restriction to use, modify and improve the code, so     *
+ * please do it and share the improvements.                             *
+ *                                                                      *
+ * Let's have Fun!!                                                     *
+ *                                                                      *
+ * ---------------------------------------------------------------------*/
+
+#if !defined(__IRRIGATION_SYSTEM_STORAGE_STORAGE_H__)
+#define __IRRIGATION_SYSTEM_STORAGE_STORAGE_H__
+#include <Arduino.h>
+#include <EEPROM.h>
+#include "../../utils/logger.h"
+
+#define BLYNK_TEMPLATE_ID_MAX     20  // Max number of characters
+#define BLYNK_TEMPLATE_NAME_MAX   20
+#define BLYNK_AUTH_TOKEN_MAX      40
+#define BLYNK_TEMPLATE_ID_BASE    0   // base address in bytes
+#define BLYNK_TEMPLATE_NAME_BASE  20
+#define BLYNK_AUTH_TOKEN_BASE     40

```

Comment:
Shouldn't be this defined on the WifiManager or on a Blynk specific header?

Reply:
The idea here is to have the ability to save credentials, maybe remove "BLYNK" from the name, but it can be used for other platform that most likely will require kind of the same data.
To have something like
0  |------------|
   |            |  <-- space to save credentials
   |            |
80 |------------|
   |            |  <-- space to save reley 1 conf
   |------------|
   |            |  <-- space to save reley 2 conf
   |------------|
   |            |
   |------------|
   |            |
   |------------|